content/300-guides/200-deployment/110-deployment-guides/400-deploying-to-aws-lambda.mdx

```diff
@@ -305,194 +247,55 @@ With the API base URL: `https://UNIQUE_IDENTIFIER.execute-api.us-east-1.amazonaw
 
-Refer to the [Serverless Webpack documentation](https://www.serverless.com/plugins/serverless-webpack) for additional configuration.
+</Admonition >
 
-</details>
+## Deploy only the required files
 
-Install the dev dependency by entering the following command:
+To reduce your deployment footprint, you can change your deployment process so that you only upload the files that your application needs. The method for doing so depends on whether you use the Serverless Framework or not.
 
-```sh
-npm install -D serverless-webpack-prisma serverless
-```
+### If you use the Serverless Framework
 
-Update your `serverless.yml` by adding the `serverless-webpack` to the plugins list.
+You can use the Serverless configuration file to [deploy only the binary that your project needs](/guides/deployment/deployment-guides/use-prisma-with-serverless-framework#reduce-the-package-footprint).
 
-```yaml file=serverless.yml
-plugins:
-  - serverless-webpack
-  - serverless-webpack-prisma
+### If you do not use the Serverless Framework
 
-custom:
-  webpack:
-    includeModules: true
-```
+If you do not use the Serverless Framework to deploy your project, you can use the following method to remove any unneeded files.
 
-`serverless-webpack-prisma` automatically runs `prisma generate` so you don't need to use the webpack script option separately as shown below.
+1. Create a zip script that only zips the files that your project needs. [Example script](https://github.com/prisma/ecosystem-tests/blob/dev/platforms-serverless/lambda/zip.sh).
 
-```yaml file=serverless.yml
-custom:
-  webpack:
-    packagerOptions:
-      scripts:
-        - prisma generate
-```
+2. Upload the zip. For example, to do this with the [AWS CLI](https://aws.amazon.com/cli/), you might use a command similar to the following:
 
-`serverless-webpack-prisma` removes the unnecessary dependencies and packages the right Prisma engine to be used in your Lambda. Therefore, delete it in the `package.patterns` configuration in your `serverless.yml` file.
+   ```terminal
+   aws lambda update-function-code --function-name my-function --zip-file "fileb://lambda.zip"
+   ```
 
-```yaml file=serverless.yml
-package:
-  patterns:
-    - '!node_modules/.prisma/client/libquery_engine-*'
-    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
-    - '!node_modules/prisma/libquery_engine-*'
-    - '!node_modules/@prisma/engines/**'
-```
+## Summary
 
-Once you've updated your `serverless.yml` file, redeploy your application by running `serverless deploy`. The deployment output will show you the correct Prisma Engine is packaged and distributed.
+Congratulations! You have successfully deployed the API to AWS Lambda.
```

Comment:
Style guide question: Do we keep using the "Congratulations!" in the **Summary**?

In terms of tone, it feels a bit more conversation and emotional, which is somewhat different from traditional docs guidelines, but perhaps something for us to discuss.

Reply:
I think it's fine for guides like this.